src/index.scss

```diff
@@ -22,65 +25,69 @@
 }
 
 .dv-ChartContainer {
-  // position: static;
+  // this is a little hacky, sorry
+  margin-bottom: -1rem;
+
+  & + .dv-ChartContainer {
+    padding-top: 0;
+    margin-bottom: 0;
+  }
 }
 
 .dv-Sidebar {
   position: absolute;
-  top: 2.85rem;
-  left: 0;
+  top: 0;
+  right: 0;
   z-index: 100;
   overflow: hidden;
-  width: 50px;
-  height: calc(100% - 4.85rem);
+  width: 0;
+  bottom: 5.5rem;
   background-color: #fff;
   box-shadow: 5px 0 40px 0 rgba(0, 0, 0, 0.2);
   transition: width 0.4s ease;
 
   will-change: width;
+
   &__button {
     position: absolute;
-    top: 2rem;
-    left: 50%;
     z-index: 1;
     margin: 0;
+    top: 1rem;
+    right: 1rem;
     padding: 1rem;
     border: none;
     background-color: transparent;
-    text-transform: uppercase;
-    letter-spacing: 0.1em;
-    font-weight: bold;
-    font-size: 14px;
     cursor: pointer;
-    transition: left 0.4s ease;
-    transform: rotate(270deg) translateX(-50%);
-    transform-origin: left;
+    transition: color 200ms;
 
     -webkit-appearance: none;
     -moz-appearance: none;
+
+    &:hover, &:focus, &:active {
+      color: #22c8a3;
+    }
+  
   }
   &__interior {
     visibility: hidden;
-    overflow-y: auto;
-    padding: 1rem;
-    padding-right: 3rem;
+    position: relative;
+    padding: 1rem 2rem 2rem;
     width: 300px;
-    height: 100%;
+    max-width: 100vw;
     opacity: 0;
     transition: opacity 0.4s ease;
   }
 }
 
-.dv-Sidebar:hover,
```

Comment:
We're keeping the sidebar all the way offscreen and expanding it with a button, instead of having the hover situation